content/docs/introduction/comparison.md

```diff
@@ -244,43 +244,39 @@ blackbox probing is sufficient.
 If you want to do whitebox monitoring, or have a dynamic or cloud based
 environment, then Prometheus is a good choice.
 
+From https://prometheus.io/docs/introduction/comparison/
+
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensuapp.org/) is broadly speaking a more modern Nagios.
+[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
 
 ### Scope
 
-The same general scope differences as in the case of
-[Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
+Sensu checks adhere to the Nagios check specification (https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html). Sensu checks operate on two different models: publish and subscribe or standalone. It is possible to silence not just individual checks, but entire subscriptions. 
 
-The primary difference is that Sensu clients [register themselves](https://sensuapp.org/docs/0.27/reference/clients.html#what-is-a-sensu-client),
-and can determine the checks to run either from central or local configuration.
-Sensu does not have a limit on the amount of perfData.
+The primary difference is that Sensu clients [register themselves](https://docs.sensu.io/sensu-core/latest/reference/clients),
```

Comment:
Difference to what?

Reply:
Good question. This is from the original comparison doc, so I've got no context as to what it was being compared against. I'm assuming it's Nagios? The intent for the change here was to point to the updated docs page. I can tweak this, though.

Reply:
It was to Nagios, but you removed that from the previous sentence.

Reply:
Ah. I'll add it back.

Reply:
As long as it reads properly.